### ISSUE TITLE
Delete Previous Shapes

### ISSUE THREAD

**Issue by mustafaulukaya** [joelmoniz/Shape-Sketch#7]
The shapes are not erased. Deletion not found. Either no, or it's hard to find. There should be a "clear" button in a clear place. It is necessary to close the program completely. There must be a deletion before the Shape-Sketch window closes.

**Comment by joelmoniz** [comment]
Indeed, there currently isn't a delete button, as reported in #2 and #7
